.storybook/addons/usage/register.js

```diff
@@ -0,0 +1,184 @@
+
+    for (let index = 1; index < componentName.length; index++) {
+      const char = componentName.charAt(index);
+
+      if (char === '-') {
+        index++;
+        result = result + componentName.charAt(index).toUpperCase();
+      } else {
+        result = result + char;
+      }
+    }
+
+    return result;
+  };
+
+  return {
+    PURE_HTML_TAG: pureHTML,
+    REACT_TAG: formatReactTag(pureHTML),
+  };
+};
+
+const beginsWithVowel = (componentName = '') => {
+  const vowels = ['a', 'e', 'i', 'o', 'u'];
+  const firstLetter = componentName.substr(0, 1);
+  let beginsWithVowel = false;
+
+  for (let index = 0; index < vowels.length; index++) {
+    if (firstLetter === vowels[index]) {
+      beginsWithVowel = true;
+      break;
+    }
+  }
+
+  return beginsWithVowel;
+};
+
+const MyPanel = () => {
+  const param = useParameter('usage', {});
+
+  const { PURE_HTML_TAG, REACT_TAG } = formatComponentName(param.componentName);
+  const { innerHTML, propsPureHTML, propsReact, usageReact } = param;
+
+  return (
+    <div className="markdown-body" style={{ margin: '15px' }}>
+      <h2 id="usage">Usage</h2>
+      <p>
+        <strong>Important:</strong> If this component needs to run in Internet
+        Explorer 11,{' '}
+        <a href="https://github.com/axa-ch/patterns-library/tree/develop/src/components/05-utils/polyfill">
+          you need to use our polyfill
+        </a>
+        .
+      </p>
+      <pre>
+        <code className="language-bash">
+          npm install @axa-ch/{PURE_HTML_TAG}
+        </code>
+      </pre>
+      <pre>
+        <code className="language-js">
+          import '@axa-ch/{PURE_HTML_TAG}';
+          <br />
+          ...
+          <br />
+          &lt;axa-{PURE_HTML_TAG}
+          {propsPureHTML && ` ${propsPureHTML}`}&gt;{innerHTML}&lt;/axa-
+          {PURE_HTML_TAG}&gt;
+        </code>
+      </pre>
+      <h3 id="react">React</h3>
+      {usageReact ? (
+        usageReact
+      ) : (
+        <>
+          <p>
+            Create a React-ified {PURE_HTML_TAG} with the createElement function
+            from your React version and then use it like this:
+          </p>
+          <pre>
+            <code className="language-js">
+              import {'{ createElement }'} from 'react';
+              <br />
+              import createAXA{REACT_TAG}React from '@axa-ch/
+              {PURE_HTML_TAG}
+              /lib/index.react';
+              <br />
+              <br />
+              const AXA{REACT_TAG}React = createAXA
+              {REACT_TAG}React(createElement);
+              <br />
+              <br />
+              export default AXA{REACT_TAG}React;
+            </code>
+          </pre>
+          <pre>
+            <code className="language-js">
+              &lt;AXA{REACT_TAG}React{propsReact && ` ${propsReact}`}&gt;
+              {innerHTML && (
+                <>
+                  <br />
+                  {`  ${innerHTML}`}
```

Comment:
You could try to format your html code with a npm tool like https://www.npmjs.com/package/html-prettify (maybe you find a better tool with some more downloads) . So you do not need to add these empty spaces here. But you could try that in a seperate branch with a seperate issue ticket.